### ISSUE TITLE
Linnea shares workshop videos to YouTube

### ISSUE THREAD

**Issue by avidrucker** [knowledge-for-good/knowledgeforgood#75]
So we may:
1. Analyze how we may improve our instruction
2. Upload to YouTube to share with others

Edit: Updated to reflect change in task.

**Comment by avidrucker** [comment]
@seekanddefine Are these videos going onto YouTube this week?

**Comment by avidrucker** [comment]
Feel free to remove the `blocked` label, change the issue name from "w/ Avi" to "w/ the world via YouTube" , and mark closed as you see fit :wink:

**Comment by seekanddefine** [comment]
I will be working all week to get them up. Should probably try to divide it up. It takes the length of the video to export the video, so the four hour workshop videos take a long time and if there is any interruption I have to start over. This happened multiple times. Once I got the first few exported then I realized I still couldn't upload them because they were too long, but now we are good to go and the first video is up. Second one loading now.
